lib/source/metadata/parser.js

```diff
@@ -116,6 +116,7 @@ Parser.mappings = {
       return;
     }
 
+    properties['iam-role'] = roles[0].substr(path.length);
```

Comment:
@dgreene-r7 that looks so much better. 😍 The `substr` thing felt like a hack and I knew there was a more clear way to do it. Just my lack of touching javascript in a long time I guess.